pkg/state/state.go

```diff
@@ -2294,6 +2297,8 @@ func markExcludedReleases(releases []ReleaseSpec, selectors []string, commonLabe
 		if err != nil {
 			return nil, fmt.Errorf("failed to parse condition in release %s: %w", r.Name, err)
 		}
+		// reset the labels to the original
+		r.Labels = orginReleaseLabel
```

Comment:
I thought the intention of this function, `markExcludedReleases`, was to DO mutate `r.Labels`, which would mean resetting it to keep it immutable is incorrect.
But the test is passing right? 🤔

Reply:
@mumoshu  I think that the main goal of the funciton is to filter releases. it should't change the `Labels`. so all tests is passed.